boat_simulator/nodes/physics_engine/force_computation.py

```diff
@@ -0,0 +1,43 @@
+"""Class to compute lift and drag forces acting on a medium."""
+
+from typing import Tuple
+
+import numpy as np
+from numpy.typing import NDArray
+
+from boat_simulator.common.types import Scalar
+
+
+class ForceComputation:
```

Comment:
This class is specific to computing the forces on a medium -- this class name is too generic